agent/engine/docker_task_engine.go

```diff
@@ -38,6 +44,10 @@ const (
 	capabilityPrefix             = "com.amazonaws.ecs.capability."
 )
 
+var (
+	S3_REGEX = regexp.MustCompile("^s3://([^/]+)/(.*)$")
```

Comment:
The other things in this file are a bad example, but C_STYLE_CONSTANTS are discouraged in Go.  Can you change this to S3Regex if it should be public or s3Regex if it should be private?